pkg/artifactory/resource/security/resource_artifactory_group.go

```diff
@@ -151,6 +151,34 @@ func groupParams(s *schema.ResourceData) (Group, bool, error) {
 	return group, includeUsers, nil
 }
 
+func GetDataSourceGroupSchema() map[string]*schema.Schema {
```

Comment:
I just realized these changes are made to the Resource file, not the Data Source file. These should only be applied to the data source. There should be minimal changes to the resource.